### ISSUE TITLE
[BUG]? No catch for tag: bad-response, "Bad status code returned: 403"

### ISSUE THREAD

**Issue by wakatara** [abrochard/emacs-todoist#14]
I'm getting a forbidden using emacs-todoist though I'm reasonably certain I've set the api creds right and the environment variable correctly.

OSX 10.15.1
Emacs 26.3
fish 3.02 via tmux 2.9a using iTerm2

Getting the following with the api token set and a fish environment variable set
```
Contacting host: api.todoist.com:443
todoist--query: No catch for tag: bad-response, "Bad status code returned: 403"
```
I've set 
  (setq todoist-token "a67XXX") before invoking the require
I've additionally tried used the (use-package todoist... ensure and config:/init: etc) method as well. Neither seems to work though the package loads I can call Todoist via M-x todosit but then seem to get a 403 as noted in the above interface. 

I've also used setenv to set the TODOIST_TOKEN within emacs to the above api key.

Any ideas or is this an acual bug?

**Comment by wakatara** [comment]
(just a note, I am also using Todoist premium).

**Comment by abrochard** [comment]
Thank you for reaching out. Does this happen every time or is it inconsistent? 
If it happens all the time and you are setting your token with something like `(setq todoist-token "XXXXXXXXXXXXXXXXXXXX")` then your token might be wrong.

**Comment by wakatara** [comment]
Hey @abrochard ! 
(wow, thanks for getting back to me so quickly!)

I am not sure how as I checked it twice, but I reissued the api token and tried again and it works fine now. I apologize for bugging you. It serves me right for working on installing new stuff after midnight... =p 

Thanks again for the quick response and what is looking like a super useful lib!
(and if you're in America and celebrating - happy thanksgiving!)
Daryl.
